app/src/main/res/values/strings.xml

```diff
@@ -34,5 +34,7 @@
     <string name="title_event_detail">Event Detail</string>
     <string name="subtitle_organizer">Organizer: %s</string>
     <string name="event_thumbnail">Event Thumbnail</string>
+    <string name="text_empty">Nothing to show here</string>
```

Comment:
Binding a text which is constant seems obsolete. We'll make messages more user-friendly when the app is feature ready. Currently, we show 404, timeout without any modification and with no data validation :smile: All of that will be changed soon